egs/lrs/asr1/clean.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
Do we need this?

Reply:
As @b-flo  said, we can delete this clean.sh file